backend/src/v5/processors/teamspaces/projects/models/containers.js

```diff
@@ -55,4 +56,18 @@ Containers.getContainerStats = async (teamspace, project, container) => {
 	};
 };
 
+Containers.getRevisions = async (teamspace, container, showVoid) => {
+	const revisions = await getRevisions(teamspace, container, showVoid,
+		{ _id: 1, author: 1, timestamp: 1, tag: 1, void: 1 });
+
+	return revisions.map((revision) => ({ ...revision, _id: UUIDToString(revision._id) }));
+};
+
+Containers.updateRevisionStatus = async (teamspace, container, revision, status) => {
+	if (typeof status === 'boolean') {
```

Comment:
we shouldn't be doing type checking here. Type checks should happen in the middleware.
![image](https://user-images.githubusercontent.com/11945337/133470948-a64a2ccd-c4ea-4892-ad70-8030a6139bfe.png)

In essence anything entering processor, we assume is type checked and validated.

Also, if the type check failed we should be throwing an error, as it stands you'd be returning a response 200 telling the user you've done the request - which will not be correct